portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewSorting.testcase

```diff
@@ -16,27 +16,26 @@ definition {
 		task ("When Create a DataSet ") {
 			DataSetAdmin.goToDataSetAdminPage();
 
-			DataSetAdmin.createDataSet(
-				key_name = "Test Data Set",
-				key_type = "/c/fdssorts");
-
-			Refresh();
-		}
-
-		task ("When go to the View page of the data set created.") {
-			DataSetAdmin.goToViews(dataSetName = "Test Data Set");
+			JSONDataSet._createDataSetEntry(
+				dataSetName = "Test Data Set",
+				restApplication = "/c/fdssorts",
+				restEndpoint = "/",
+				restSchema = "FDSSort");
 		}
 
 		task ("And adding a data set view") {
-			DataSetAdmin.createDataSetView(
-				description = "View Description",
-				key_name = "View Test");
+			JSONDataSet._createFDSViewViaAPI(
+				dataSetName = "Test Data Set",
+				dataSetViewDescription = "View Description",
+				dataSetViewName = "View Test");
```

Comment:
I think this should be named JSONDataSet._createDataSetViewViaAPI

Reply:
as I didn't create this macro and it affects a lot of data set tests I think I'll create a PR to adjust it and update the other tests that use it.